pyinfra/modules/yum.py

```diff
@@ -41,6 +50,16 @@ def repo(
     + description: optional verbose description
     + gpgcheck: whether set ``gpgcheck=1``
     + gpgkey: the URL to the gpg key for this repo
+
+    Example:
+
+    .. code:: python
+
+        yum.repo(
+            {'Add the Docker CentOS repo'},
+            name='DockerCE',
+            baseurl='https://download.docker.com/linux/centos/docker-ce.repo',
```

Comment:
This should be a proper base URL ie `https://download.docker.com/linux/centos/7/$basearch/stable`.

Reply:
Thanks. Will fix.